### PR TITLE
add a script to bump bugs, and more comments

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

File: scripts/bump.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+import json
+import os
+import sys
+import urllib
+
+import requests
+
+# A script that bumps all bugs in a given target milestone on to the
+# next target milestone. Both milestones need to be specified, for
```
Comment (fwenzel) [inline scripts/bump.py]: Here you say they both need to be specified, but below you allow "end" to be undefined. Why?

Comment (fwenzel) [inline scripts/bump.py]: I guess without "end" you'll get a list of bugs in the current milestone with no changes made. Neat. Might mention that here.

**Comment by fwenzel** [comment]
r+wc

File: scripts/bump.py (hunk 1 above)
Comment (andymckay) [inline scripts/bump.py]: well caught

**Pull request merged**